tools/pd-ctl/pdctl/command/config_command.go

```diff
@@ -507,31 +507,30 @@ func putPlacementRulesFunc(cmd *cobra.Command, args []string) {
 		cmd.Println(err)
 		return
 	}
-	var rules []*placement.Rule
-	if err = json.Unmarshal(content, &rules); err != nil {
+
+	var opts []*placement.RuleOp
+	if err = json.Unmarshal(content, &opts); err != nil {
 		cmd.Println(err)
 		return
 	}
-	for _, r := range rules {
-		if r.Count > 0 {
-			b, _ := json.Marshal(r)
-			_, err = doRequest(cmd, rulePrefix, http.MethodPost, WithBody("application/json", bytes.NewBuffer(b)))
-			if err != nil {
-				fmt.Printf("failed to save rule %s/%s: %v\n", r.GroupID, r.ID, err)
-				return
-			}
-			fmt.Printf("saved rule %s/%s\n", r.GroupID, r.ID)
+
+	validOpts := opts[:0]
+	for _, Op := range opts {
```

Comment:
Maybe using `op` instead of `Op` will be more in line with the Go's official standard?